src/t2iapi/metric/service.proto

```diff
@@ -99,4 +99,14 @@ service MetricService {
    */
   rpc IsComputerControlled (BasicHandleRequest)
       returns (t2iapi.metric.IsComputerControlledResponse);
+
+  /*
+  Provide pm:MetricValue/@Value or pm:MetricValue/@Samples for the metric with the requested handle.
+
+  The goal of this rpc is to have values provided, they must not be stable to the same value (e.g. for measurements).
```

Comment:
must not translated -> "darf nicht"

I think this is not the intention
maybe write something like:

The goal of this rpc is to have values continually provided, whereby the values may change over time.

Reply:
```suggestion
  The goal of this rpc is to have values present continuously, the values themselves may change over time.
```